### ISSUE TITLE
[Request] QField Slots

### ISSUE THREAD

**Issue by hurrycaner** [quasarframework/quasar#1962]
I want to add links and small icons with tooltips at helper section of a QField, to do that I need a helper slot at the component.

**Comment by rstoenescu** [comment]
Will do.

**Comment by rstoenescu** [comment]
Example:
```html
<q-field
  icon="cloud"
  :error="error"
  :warning="warning"
  :label-width="3"
>
  Slot test

  <span slot="label">Slot label</span>
  <span slot="error-label">Slot error label</span>
  <span slot="warning-label">Slot warning label</span>
  <span slot="helper">Slot helper</span>
</q-field>
```
Available in 0.15.11

**Comment by gabrielmsd** [comment]
Have you created a slot for the q-field icon property?
